native/cocos/renderer/pipeline/PipelineSceneData.cpp

```diff
@@ -66,23 +66,16 @@ void PipelineSceneData::activate(gfx::Device *device) {
 }
 
 void PipelineSceneData::destroy() {
-    for (const auto &pair : _shadowFrameBufferMap) {
-        pair.second->destroy();
```

Comment:
Framebuffer::destroy() should be explicitly called. It will not be called in destructor.

Reply:
The sub class's destructor(for example, MTLFrameBuffer) will invoke destroy().

Reply:
Ah, yes. I see.